doc/userguide/install.rst

```diff
@@ -28,6 +28,7 @@ This will install Suricata into ``/usr/local/bin/``, use the default
 configuration in ``/usr/local/etc/suricata/`` and will output to
 ``/usr/local/var/log/suricata``
 
+You can obtain the source code for the latest release [from Github](https://github.com/OISF/suricata/releases/latest)
```

Comment:
Wondering if in our userguide we should point folks to GH or to our Downloads page... (https://suricata.io/download/)

Invoking master @jasonish to judge. :P

Reply:
Downloads page.  Those releases on GitHub aren't our actual releases. They're just an archive of the git repository, so no `./configure`, no `libhtp`, no `suricata-update`.

Reply:
Thanks :)

I can build upon this contribution to include that, if that's easier for everyone :)

Reply:
@jufajardini if there's anything I can do, let me know- otherwise it's all yours :)

Reply:
> @jufajardini if there's anything I can do, let me know- otherwise it's all yours :)

As per Jason's answer, instead of pointing users to our GH repo, it's best to link our Downloads page: https://suricata.io/download/

If you'd be up to change that, it's very welcome, but if not, I can do it ^^

Reply:
Changes updated.

@jasonish , you'll notice the line I added with the link to the downloads page is the longest line in the file, and does not stick with the "short lines" convention you've been using. I posit that breaking it across lines would be ugly, with very different line lengths between the line with the URL and the line above/below it.

Also, I preserved the link to Github because, for those of us hoping to find the latest available version programmatically, even if we are not downloading from GH. That link - `/releases/latest` - is the simplest, quickest, safest way I know to find the latest release of something programmatically (curl it - it's super simple JSON), and since I'm already doing it for many other softwares, I already have tooling set up to handle it. That being said, let me know if you'd like to see something different - my use case today is a Chef cookbook that I'll feed to Packer to build multi-arch AWS AMIs and vmware images.

Reply:
Actually now I'm seeing that further down on the page in question, § 3.1.3 (Compilation) says:

```shell
./scripts/bundle.sh
./autogen.sh
./configure # you may want to add additional parameters here
# ./configure --help to get all available parameters
make -j8 # j is for paralleling, you may de/increase depending on your CPU
make install # to install your Suricata compiled binary
```

Indeed, when I download the source from GH, I get `scripts/bundle.sh` (obviously), and when I download from the Suricata downloads page, I have a `libhtp` and `suricata-update` directory, and no `scripts/bundle.sh`.

Let me know how you want to proceed- I could also see that, if you want to clarify § 3.1.3, that could go in a separate PR.

Reply:
As this is the user docs, we should not use the downloads from GitHub.  Users should not need to run autogen.sh, bundle.sh.

Reply:
We have this other build guide for those wanting more cutting edge stuff: https://docs.suricata.io/en/latest/devguide/codebase/installation-from-git.html

Reply:
Okay. Are you happy with the code that's in there now? Remember, since this is a thread, the code you're seeing at the top of the thread is not the latest code.